Source/WebCore/svg/SVGElement.cpp

```diff
@@ -900,7 +900,7 @@ CSSPropertyID SVGElement::cssPropertyIdForSVGAttributeName(const QualifiedName&
     case AttributeNames::font_size_adjustAttr:
         return CSSPropertyFontSizeAdjust;
     case AttributeNames::font_stretchAttr:
-        return CSSPropertyFontStretch;
+        return CSSPropertyFontWidth;
```

Comment:
Can you add font-width here: https://searchfox.org/wubkat/rev/b36cbce69fddb7da33823f316bd8ead5bebee970/Source/WebCore/svg/svgattrs.in#62 ?

and also add a mapping for font-width here ?